spec/controllers/users_controller_spec.rb

```diff
@@ -16,21 +16,6 @@
       expect(response.code).to eq "200"
       expect(assigns[:user]).to be_new_record
     end
-
-    context "ENV has disabled sign-ups" do
-      before do
-        allow(ENV).to receive(:[]).and_return(nil)
```

Comment:
Not needed now, but the [climate control] gem is great for setting `ENV` in test.

[climate control]: https://github.com/thoughtbot/climate_control